packages/cypress-gatsby/commands.js

```diff
@@ -6,11 +6,13 @@ Cypress.Commands.add(`getTestElement`, selector =>
   cy.get(`[data-testid="${selector}"]`)
 )
 
+const TIMEOUT = 9999
+
 Cypress.Commands.add(
   `waitForAPI`,
   { prevSubject: `optional` },
```

Comment:
I'm not sure if this is right place for allowing configuration, but idea itself sounds ok - I'll take a look implementing that :)

Reply:
[Second arg is options](https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax) :) 

Also tangential to this PR. I was just wanting this functionality in my e2e development runtime tests! Only since you're making changes I thought it could make sense to make this improvement!

Reply:
> Pass in an options object to define the implicit behavior of the custom command.

I need to use callback args, and not Command registration args ;) In any case - I implemented options stuff and actually use it to timeout faster for my tests, so I'll put timeout related comments as resolved